3-box-model/styles.css

```
@@ -0,0 +1,42 @@
+header {
+  position: sticky;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0px;
+  border: 2px solid rgb(0, 255, 85);
+  color: aliceblue;
+  width: 100vw;
+  height: 60px;
+  text-align: center;
+  background-color: black;
+}
+
+.wrapper {
+  display: flex;
+}
+
+nav {
+  position: absolute;
+  width: 200px;
+  border: 2px solid blue;
+  z-index: -1;
+}
+
+section {
+  border: 2px solid red;
+  margin-inline-start: 200px;
+}
```

```suggestion
section {
  border: 2px solid red;
  left: 200px;
  position: relative;
}
```
This is what the exercise meant to be